packages/client/src/generation/TSClient/Args.ts

```diff
@@ -144,10 +144,10 @@ export type ${modelArgName} = ${baseTypeName}
 
 export class MinimalArgsType implements Generatable {
   constructor(
-    protected readonly args: DMMF.SchemaArg[],
-    protected readonly type: DMMF.OutputType,
-    protected readonly action?: DMMF.ModelAction,
-    protected readonly collector?: ExportCollector,
+    private readonly args: DMMF.SchemaArg[],
```

Comment:
See no reasons for them to be `protected`, nobody extends this class.

Reply:
I don't have an opinion on protected vs private here but would like to see this sort of change be its own commit with a clear commit message.